main.js

```diff
@@ -0,0 +1,33 @@
+const tweeter = Tweeter()
+const renderer = Renderer()
+const posts = tweeter.getPosts()
+
+renderer.renderPosts(posts)
+
+
+$('#post').on('click', function () {
+    tweeter.addPost($("#input").val())
+    renderer.renderPosts(posts)
+});
+
+$('#posts').on('click', '.delete', function () {
+    const post = $(this).closest($('.post'))
+    tweeter.removePost(post.data().post)
+    renderer.renderPosts(posts)
+})
+
+$('#posts').on('click', '.asi', function () {
+    const postId = $(this).closest($('.post')).data().post
+    const commentText = $(this).closest($('.post')).find('.marom').val()
```

Comment:
I wonder where you got the "marom" class name.... ;) anyways, in general, name the class names with a meaningful name